recipe/build.sh

```diff
@@ -2,11 +2,8 @@
 
 export ESMFMKFILE=${PREFIX}/lib/esmf.mk
 
-ESMPY_SRC=${SRC_DIR}/src/addon/ESMPy
-cd ${ESMPY_SRC}
+cd ${SRC_DIR}/src/addon/ESMPy
 
 
 ${PYTHON} setup.py build --ESMFMKFILE=${ESMFMKFILE}
 ${PYTHON} setup.py install --record record.txt
-${PYTHON} setup.py test || exit 1
```

Comment:
I eliminated the testing with the source code, it relies on `ESMFMKFILE` that is defined with the build prefix.
I added a small "real case" test instead. (Copied it from the docs.)